### ISSUE TITLE
Document `browser_action` and `page_action` or document `action` (Manifest v3) in the manifest-schema

### ISSUE THREAD

**Issue by mukul-p** [w3c/webextensions#50]
should we document browser_action and page_action or document the Manifest v3 action

**Comment by Techdojo** [comment]
I think we should do both but tag them (and any other MV2 / MV3 specific sections) as appropriate.

**Comment by dotproto** [comment]
I think it's probably best to specify all 3: `browser_action` and `page_action` for Manifest V2 and `action` for Manifest V3. 

More broadly, though, we should probably discuss how our work integrates with Manifest V2, Manifest V3, and whatever may come after that. The Chrome team is primarily interested in pursuing the future of the browser extension platform, but also recognizes that the future is built on the platform that we have today.

To that end our preference is to specify both Manifest V2 and Manifest V3 features, but to limit Manifest V2 inclusions to capabilities that a) have an equivalent in Manifest V3 and b) are common across Manifest V2 implementations. For example, this would mean that when specifying Manifest V2 capabilities we would omit both Chrome's [system.display API](https://developer.chrome.com/docs/extensions/reference/system_display/) and Firefox's [search API](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/search).

**Comment by zombie** [comment]
I agree with Simeon, and along the same lines we should treat MV3 features similarly: for now document those that a) have V2 equivalents, and b) once there's a at least a second broadly compatible implementation. In this concrete example, this would include the `action` property, implemented in Chrome and (experimentally, behind a pref) [in Firefox 91](https://bugzilla.mozilla.org/show_bug.cgi?id=1706398) to be released next week. :)

As we progress along with those specifications and implementations, a) would matter less and b) should become the main criteria.
